examples/rmg/minimal_surface/input.py

```diff
@@ -1,67 +1,125 @@
 # Data sources
 database(
-    thermoLibraries = ['primaryThermoLibrary'],
-    reactionLibraries = [],
+    thermoLibraries=['surfaceThermoPt111', 'primaryThermoLibrary', 'thermo_DFT_CCSDTF12_BAC','DFT_QCI_thermo'], # 'surfaceThermoPt' is the default. Thermo data is derived using bindingEnergies for other metals 
+    reactionLibraries = [('Surface/CPOX_Pt/Deutschmann2006_adjusted', False)], # when Ni is used change the library to Surface/Deutschmann_Ni 
     seedMechanisms = [],
     kineticsDepositories = ['training'],
-    kineticsFamilies = 'default',
+    kineticsFamilies = ['surface', 'default'],
     kineticsEstimator = 'rate rules',
 )
-
-# List of species
+catalystProperties(
+    bindingEnergies = {
+        'H': (-2.75368,'eV/molecule'),
+        'C': (-7.02516,'eV/molecule'),
+        'N': (-4.63225,'eV/molecule'),
+        'O': (-3.81153,'eV/molecule'),
+    },
+    surfaceSiteDensity=(2.483e-9, 'mol/cm^2'),
+    coverageDependence=False
+)
 species(
-    label='ethane',
+    label='CH4',
     reactive=True,
-    structure=SMILES("CC"),
+    structure=SMILES("[CH4]"),
```

Comment:
Does this SMILES need to be bracketed?

Reply:
changed it to the more standard form "C"